app/helpers/application_helper.rb

```diff
@@ -31,4 +31,61 @@ def flash_messages(opts = {})
     nil
   end
 
+  def us_states_options
```

Comment:
Does this need to include any territories, etc?

Reply:
@rsl just the US states!